indico/modules/legal/forms.py

```diff
@@ -26,5 +28,11 @@
 class LegalMessagesForm(IndicoForm):
     network_protected_disclaimer = TextAreaField(_("Network-protected information disclaimer"), widget=CKEditorWidget())
     restricted_disclaimer = TextAreaField(_("Restricted information disclaimer"), widget=CKEditorWidget())
-    tos = TextAreaField(_("Terms and conditions"), widget=CKEditorWidget())
-    privacy_policy = TextAreaField(_("Privacy Policy"), widget=CKEditorWidget())
+    tos_url = URLField(_('URL'), [URL()],
```

Comment:
why are you using `'` here and `"` for other strings?

Reply:
didn't bother changing the existing ones